lib/internal/repl.js

```diff
@@ -19,11 +20,11 @@ function createRepl(env, opts, cb) {
     cb = opts;
     opts = null;
   }
-  opts = opts || {
+  opts = util._extend({
```

Comment:
I'm curious why you wouldn't just use `Object.assign` here. Looking at the source of `util._extend` it looks to be identical. What am I missing?